app/views/assignments/rotation_generator.haml

```diff
@@ -1,20 +1,30 @@
 %h1.title Generate rotation
 
 = form_tag generate_rotation_roster_assignments_path(@roster) do
-  = label_tag :start_date
-  = text_field_tag :start_date,
-    @start_date,
-    class: 'datepicker'
-  = label_tag :end_date
-  = text_field_tag :end_date,
-    @start_date + 3.months,
-    class: 'datepicker'
-  = label_tag :user_ids, 'Users'
-  = select_tag :user_ids,
-    options_from_collection_for_select(@users, :id, :last_name,
-                                       @users.pluck(:id)),
-    multiple: true, size: 10
-  = label_tag :starting_user_id, 'Starting with'
-  = select_tag :starting_user_id,
-    options_from_collection_for_select(@users, :id, :last_name)
-  = submit_tag 'Generate rotation'
+  .container-fluid
+    .row
+      .col-xs-2
+      .col-xs-4
```

Comment:
I somehow managed to forget offset was an option for a day or two, not sure why but I'll fix it.